Explorer/Assets/Scripts/SceneRuntime/Apis/Modules/SignedFetch/SignedFetchWrap.cs

```diff
@@ -159,7 +159,17 @@ async UniTask<FlatFetchResponse> ExecuteRequestAsync()
 
                     return response;
                 }
-                catch (UnityWebRequestException e) { return new FlatFetchResponse(false, e.ResponseCode, e.ResponseCode.ToString(), e.Error, e.ResponseHeaders); }
+                catch (UnityWebRequestException e)
+                {
+                    if (e.ResponseHeaders.TryGetValue("Content-type", out var contentType) && contentType.Contains("application/json"))
```

Comment:
Just in case, make the "application/json" comparison to ignore case maybe?

Reply:
Done